### ISSUE TITLE
Fix empty key

### ISSUE THREAD

**Issue by odinuv** [keboola/generic-extractor#68]
This https://developers.keboola.com/extend/generic-extractor/configuration/config/mappings/#basic-table-mapping does not work because empty key is not accepted by docker runner

change it to something else (this is not a BC break)

**Comment by ondrejhlavacek** [comment]
Tady se to objevilo znovu - https://keboola.zendesk.com/agent/tickets/6715

**Comment by odinuv** [comment]
ok, vrhnu se na to dneska

**Comment by ondrejhlavacek** [comment]
Btw týpek to prej vyřešil...
```
Nakonec se mi to podařilo vyřešit díky zaslané dokumentaci - problém byl v 
tom, že jsem první prvek určoval jako tags[0], správné určení je ale tags.0 
:)
```

**Comment by odinuv** [comment]
fixed in ec796bd4cc8ed868dd1f792dd74b2d95185d47c3
